project_management/pm/src/components/ProjectsFolder/ProjectsList.js

```diff
@@ -5,28 +5,38 @@ import {
   TableContainer,
   TableRow,
 } from "@mui/material";
-import { NavLink } from "react-router-dom";
-import React, { useEffect, useState,useContext } from "react";
-import { getProjects } from "../../controllers/ProjectsController";
+import { NavLink, useNavigate } from "react-router-dom";
+import React, { useEffect, useState, useContext } from "react";
+import {
+  deleteProject,
+  getProjects,
+} from "../../controllers/ProjectsController";
 import { cookiesContext } from "../../App";
+import DeleteOutlineIcon from "@mui/icons-material/DeleteOutline";
 
 export default function ProjectsList() {
   const [projects, setProjects] = useState([]);
   const cookies = useContext(cookiesContext);
-  
+  const [user,] = useState(cookies.get("user"));
+  const [token,] = useState(user.token);
   useEffect(() => {
-    console.log({cookies});
-    const token = cookies.get("token");
 
     async function getData() {
-      const result = await getProjects(token);
-
+      console.log({token})
+      const result = await getProjects(user.id,token);
+      
       setProjects(result);
     }
 
     getData();
   }, []);
 
+  const handelDelete = async (id) => {
+    const responseStatus = await deleteProject(id, token);
+    console.log({responseStatus})
+    if (responseStatus === 200)  window.location.reload(); 
```

Comment:
i think i can delete the row or just recall getData to update your states instead of reload page, ask chagpt about best practice and tell me

Reply:
ok, important point